styles/styles.css

```diff
@@ -294,13 +294,11 @@ main img {
   :root {
     /* font sizes */
     --heading-font-size-xxl: 60px;
-    --heading-font-size-xl: 48px;
-    --heading-font-size-l: 36px;
-    --heading-font-size-m: 30px;
-    --heading-font-size-s: 24px;
-    --heading-font-size-xs: 22px;
-
-    /* nav */
+    --heading-font-size-xl: 36px;
+    --heading-font-size-l: 30px;
+    --heading-font-size-m: 24px;
+    --heading-font-size-s: 20px;
+    --heading-font-size-xs: 18px;/* nav */
```

Comment:
change the variable name to something like heading-nav-font

Reply:
Done